src/docs/references/page-schema/index.md

```diff
@@ -0,0 +1,1097 @@
+---
+title: Page schema
+slug: page-schema
+updated: 2023-02-15
+category: references
+ingress:
+  Reference documentation providing information on the page schema.
+published: true
+---
+
+> how the data looks like after people add fields and then remove them -
+> aka how the deleted data looks like in the fetched asset, as that’s
+> causing work for me. - And we should probably document those somewhere
+> for customizers.
+
+// TODO do we show the \_editor attributes as well?
```

Comment:
We could host and upload to [this file](https://github.com/sharetribe/sharetribe-custom/blob/master/resources/json_schema/pageSchema.json) as a gist for people who want to view the whole schema

Reply:
That's a good idea!